src/main/java/com/woowacamp/soolsool/core/liquor/repository/LiquorQueryDslRepository.java

```diff
@@ -55,13 +55,27 @@ public List<LiquorElementResponse> getCachedList(
         final Pageable pageable
     ) {
         log.info("LiquorQueryDslRepository getCachedList");
-        return getList(
-            LiquorSearchCondition.nullObject(),
-            pageable,
-            null
-        );
+        return getList(pageable, null);
     }
 
+    private List<LiquorElementResponse> getList(
+        final Pageable pageable,
+        final Long cursorId
+    ) {
+        return queryFactory.select(
+                Projections.constructor(
+                    LiquorElementResponse.class,
+                    liquor
+                )
+            )
+            .from(liquor)
+            .where(cursorId(cursorId))
+            .orderBy(liquor.id.desc())
+            .limit(pageable.getPageSize())
+            .fetch();
+    }
```

Comment:
```suggestion
    private List<LiquorElementResponse> getFirstPageList(
        final Pageable pageable
    ) {
        return queryFactory.select(
                Projections.constructor(
                    LiquorElementResponse.class,
                    liquor
                )
            )
            .from(liquor)
            .orderBy(liquor.id.desc())
            .limit(pageable.getPageSize())
            .fetch();
    }
```

1. where절이 항상 null이어서 관련된 매개변수를 제거해봤습니다.
2. 좀 더 직관적으로 이름을 바꿔봤습니다. public 메서드와 private 메서드의 이름을 같게하는게 거슬렸는데 어떻게 생각하시나요?
3. 첫 페이지 쿼리를 QueryDsl이 아니라 Jpa를 이용하면 더 가독성이 좋을 것 같은데 어떻게 생각하세요?

Reply:
1. 좋네요
2. 좋아요
3. 저는 동일한 조회역할을 하는 부분이 jpa, querydsl에 있으면 더 헷갈릴거같다고 생각이 드는데 어떻게 생각하시나요??